en/docs/remote-apps/connect-remote-app.md

```diff
@@ -0,0 +1,120 @@
+
+To set up the Ballerina Project:
+
+1. Create a directory in your machine (e.g., named `HelloWorld`) and move your file with the remote ballerina application (in this example the `hello_service.bal` file) to it.
+
+    !!! info
+        This directory will be referred to as `<BALLERINA_PROJECT>`from here onward.
+
+2. To link the Ballerina application to Choreo via the application secret, open a new file in a text editor and add the Ballerina configurations you copied in Step 1. Then save it as `Config.toml` in the `<BALLERINA_PROJECT>`.
+
+3. To enable the Ballerina application to publish observability statistics, save a file named `Ballerina.toml` with the following content in the `<BALLERINA_PROJECT>`.
+
+    ```ballerina
+    [build-options]
+    observabilityIncluded = true
+    ```
+   
+4. For the Ballerina application to import the `ballerinax/choreo` module, open the `.bal` file (in this example, `hello_service.bal`) and at the following line at the top.
+
+    ```ballerina
+    import ballerinax/choreo as _;
+    ```
+   
+Now you have completed setting up your Ballerina project. It will connect to Choreo once you build and run it.
+
+## Step 3: Build the Ballerina project
+
+In this step, you will build and run your Ballerina project. 
+
+1. To build the Ballerina project, open your terminal, navigate to `<BALLERINA_PROJECT>`, and issue the following command.
+
+    ```text
+    bal build
+    ```
+   
+    This creates a `JAR` in the `<BALLERINA_PROJECT>/target/bin` sub directory.
+   
+2. To run the Ballerina application, issue the following command from the `<BALLERINA_PROJECT>`.
+
+    ```
+    bal run target/bin/<JAR_NAME>.jar
+    ```
+   
+    e.g., `bal run target/bin/HelloService.jar`
+    
+    Logs similar to the following appear in the terminal.
+    
+    ```
+    ballerina: initializing connection with observability backend periscope.choreo.dev:443
+    ballerina: visit http://console.choreo.dev/observe/app/91e9750f-e2e1-11eb-90ce-de6d84510939/dd9780a8-e2e2-11eb-90ce-de6d84510939 to access observability data
+    ballerina: started publishing metrics to Choreo
+    [ballerina/http] started HTTP/WS listener 0.0.0.0:9090
+    ballerina: started publishing traces to Choreo
+    ```
+   
+    These logs indicate that your Ballerina application is connected to Choreo and that it is publishing data to it. Now you can invoke it and generate some observability data.
+   
+## Step 4: Invoke the Ballerina application
+
+To invoke your Ballerina application, send a few requests.
+
+In this example, you can invoke the `HelloService` application via the following cURL command.
+
+```text
+curl http://localhost:9090/hello/sayHello
+```
+
+After sending a few requests, you can proceed to the next step to view the observability statistics generated for your Ballerina application.
+
+## Step 5: View observability statistics
```

Comment:
To view the Observability stats, User could also click on the link provided in the terminal as given in the sample in Step 3. 
```
ballerina: visit http://console.choreo.dev/observe/app/91e9750f-e2e1-11eb-90ce-de6d84510939/dd9780a8-e2e2-11eb-90ce-de6d84510939 to access observability data
```
Maybe we could add that approach under Step 5 as well.

Reply:
Done